lib/OpenQA/WebAPI/Controller/API/V1/Bug.pm

```diff
@@ -62,10 +62,16 @@ Note: Only one of "refreshable" and "created_since" can be used at the same time
 sub list {
     my ($self) = @_;
 
+    my $validation = $self->validation;
+    $validation->optional('refreshable')->num(0, 1);
+    $validation->optional('delta')->num(0);
+    $validation->optional('created_since')->num(0);
```

Comment:
Hopefully nobody is passing non-integer numbers here (which previously likely worked).

Reply:
Well, it would have resulted in a database error after subtracting said string from the current time.